src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/service/FlowGenericCarrier.java

```diff
@@ -58,4 +59,10 @@ default void sendActivateFlowMonitoring(RequestedFlow flow) {}
     default void sendDeactivateFlowMonitoring(String flow, SwitchId srcSwitchId, SwitchId dstSwitchId) {}
 
     void sendInactive();
+
+    /**
+     * Sends UpdateFlowInfo to flow-monitoring topology.
+     * @param flowInfo message to send
+     */
+    default void sendNotifyFlowMonitor(UpdateFlowInfo flowInfo) {}
```

Comment:
What is the reason to have empty default implementation? Why not require from an every flow operation to notify the monitoring?